mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/mapping/definition/programmatic/impl/IndexingDependencyMappingContextImpl.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.v6poc.entity.pojo.mapping.definition.programmatic.impl;
+
+import org.hibernate.search.v6poc.entity.pojo.dirtiness.ReindexOnUpdate;
+import org.hibernate.search.v6poc.entity.pojo.extractor.ContainerValueExtractorPath;
+import org.hibernate.search.v6poc.entity.pojo.mapping.building.spi.PojoMappingCollector;
+import org.hibernate.search.v6poc.entity.pojo.mapping.building.spi.PojoMetadataContributor;
+import org.hibernate.search.v6poc.entity.pojo.mapping.definition.programmatic.IndexingDependencyMappingContext;
+import org.hibernate.search.v6poc.entity.pojo.mapping.definition.programmatic.PropertyMappingContext;
+import org.hibernate.search.v6poc.entity.pojo.model.additionalmetadata.building.spi.PojoAdditionalMetadataCollectorPropertyNode;
+
+
+/**
+ * @author Yoann Rodiere
+ */
+public class IndexingDependencyMappingContextImpl
+		extends DelegatingPropertyMappingContext
+		implements IndexingDependencyMappingContext,
+		PojoMetadataContributor<PojoAdditionalMetadataCollectorPropertyNode, PojoMappingCollector> {
+
+	private ContainerValueExtractorPath extractorPath = ContainerValueExtractorPath.defaultExtractors();
+	private ReindexOnUpdate reindexOnUpdate = ReindexOnUpdate.USED;
```

Comment:
Note to myself: this should be null, and handled accordingly in `contributeModel`.

Reply:
Done